src/main/java/com/minecolonies/coremod/client/render/EmptyTileEntitySpecialRenderer.java

```diff
@@ -8,23 +8,9 @@
  */
 public class EmptyTileEntitySpecialRenderer extends TileEntitySpecialRenderer<TileEntityColonyBuilding>
 {
-
-    /**
-     * {@inheritDoc}
-     * Method is empty because there are no special ways required to render.
-     *
-     * @param tileEntity   Tile entity to render
-     * @param x            X-coordinate
-     * @param y            Y-coordinate
-     * @param z            Z-coordinate
-     * @param partialTicks probably used for animations
-     * @param destroyStage don't know, doesn't really matter
-     */
     @Override
-    public void renderTileEntityAt(final TileEntityColonyBuilding tileEntity, final double x, final double y, final double z, final float partialTicks, final int destroyStage)
+    public void render(final TileEntityColonyBuilding te, final double x, final double y, final double z, final float partialTicks, final int destroyStage, final float alpha)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this method to simply inherit it. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1185)